gateway-server/src/main/resources/bootstrap.yml

```diff
@@ -32,6 +32,15 @@ spring:
             - Path=/api/roles/**
           filters:
             - StripPrefix=1
+      globalcors:
+        corsConfigurations:
+          '[/**]':
+            allowedOrigins: "*" # TODO : limit origins - or reconsider CORS approach?
```

Comment:
Maybe it is worth adding a technical story for that...

Reply:
Good idea - I will raise a Jira ticket